css/css-grid-1/implicit-grids/grid-support-grid-auto-columns-rows-002.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>CSS Grid Layout Test: Support for 'grid-auto-columns' and 'grid-auto-rows' accepting track listing as value</title>
+        <link rel="help" href="http://www.w3.org/TR/css-grid-1/#auto-tracks">
+        <link rel="match" href="../reference/grid-support-grid-auto-columns-rows-002-ref.html">
+        <link rel="author" title="Rachel Andrew" href="mailto:me@rachelandrew.co.uk"/>
+        <style>
+         #grid {
+             display: grid;
+             grid-auto-columns: 25px 50px;
+             grid-auto-rows: 25px 50px;
```

Comment:
Probably it'd be nice to use different values for `grid-auto-rows` than `grid-auto-columns` (e.g. `grid-auto-rows: 40px 30px`). That way we might catch some implementation issue if it's mixing both properties.
Of course this will imply that you need to modify the reference too.